src/coreclr/vm/corelib.h

```diff
@@ -694,6 +694,10 @@ DEFINE_METHOD(MEMORY_MARSHAL,       GET_ARRAY_DATA_REFERENCE_MDARRAY, GetArrayDa
 DEFINE_CLASS(INTERLOCKED,           Threading,              Interlocked)
 DEFINE_METHOD(INTERLOCKED,          COMPARE_EXCHANGE_T,     CompareExchange, GM_RefT_T_T_RetT)
 DEFINE_METHOD(INTERLOCKED,          COMPARE_EXCHANGE_OBJECT,CompareExchange, SM_RefObject_Object_Object_RetObject)
+DEFINE_METHOD(INTERLOCKED,          COMPARE_EXCHANGE_BYTE,  CompareExchange, SM_RefByte_Byte_Byte_RetByte)
+DEFINE_METHOD(INTERLOCKED,          COMPARE_EXCHANGE_SHRT,  CompareExchange, SM_RefShrt_Shrt_Shrt_RetShrt)
```

Comment:
Nit: UShort/Short (and SByte/Byte) have signed wrapper and unsigned actual implementation, so it would be better to use UShort here.

Reply:
I see that now and will revert the ushort changes... but why do we differ like between short and int/long?

Reply:
Historically, we used the signed actual implementation and unsigned wrapper everywhere.

It was flipped in https://github.com/dotnet/runtime/pull/99011/ for small integer types. The fallback implementation for unsigned types is a bit more natural to write (no need to worry about sign-extensions during bit twiddling).